### PR TITLE
Update molecular data submission with memory and docker platform settings

### PR DESCRIPTION
**Pull request by guanqiaofeng**
Updated submitting-molecular-data.md documentation for  
1) local runtime environment (required memory exceeds available memory) and  
2) docker platform variable for Apple M1 users

### CODE REVIEW

**Comment by guanqiaofeng** [comment]
https://github.com/icgc-argo/argo-docs/pull/592

File: docs/submission/submitting-molecular-data.md [hunk 1]
```diff
@@ -329,6 +329,8 @@ nextflow run main.nf \
 :::note
 
 - Field `path` in table `files.tsv` is `Required` for local data and it will have to be formatted using the file path **relative** to the directory you run the data submission workflow
+- If data is submitted through a laptop, users may come across issue that required memory exceeds available memory. In such case, add `--score_mem 16 --score_cpus 2` in the nextflow command. The required memory and cpu use can be adjusted in these arguments.
```
Comment (edsu7) [inline docs/submission/submitting-molecular-data.md:332]: `users may come across issue that required memory exceeds available memory`
to
`users may encounter hardware resource limitations`

To be less verbose

File: docs/submission/submitting-molecular-data.md [hunk 2]
```diff
@@ -329,6 +329,8 @@ nextflow run main.nf \
...
+- If local computer has Mac M1 (ARM64) system, you may run into docker platform issue. To solve this issue, users can add `export DOCKER_DEFAULT_PLATFORM=linux/amd64` before executing the nextflow pipleline.
```
Comment (edsu7) [inline docs/submission/submitting-molecular-data.md:333]:
```
To run docker on ARM64 systems,  users can execute `export DOCKER_DEFAULT_PLATFORM=linux/amd64` in the terminal before executing the nextflow pipleline
```

**Review by edsu7 — CHANGES_REQUESTED** [review]

**Comment by guanqiaofeng** [comment]
updated based on Edmund's suggestions

**Review by edsu7 — APPROVED** [review]
LGTM!

**Pull request merged**